frontend/ui/widget/titlebar.lua

```diff
@@ -56,6 +56,8 @@ local TitleBar = OverlapGroup:extend{
     right_icon_tap_callback = function() end,
     right_icon_hold_callback = function() end,
     right_icon_allow_flash = true,
+        -- set any of these _callback to false to not handle the event
+        -- and let it propagate; otherwise the event is discarded
```

Comment:
Indent on purpose?

Reply:
To emphasize that the comment concerns **above** code, I think.

Reply:
> To emphasize that the comment concerns above code, I think.

Yep, that's exactly how I usually use/do it.

Reply:
In this case a comment at the top would make more sense to me, but /shrug

Reply:
For me, comments above are generic/explaining.
Comments at end of line can be that too, but I may also add some to explain a quick edge case or specific that is not worth being in the generic comment above, as distracting or too edge casy.
So, when that additional edgecasey comment is too long to be put at end of line, I do the trick with indentation below.

Reply:
I just find it harder to follow. Instead of "here's what this is" it's "here's what this was" which works great for a single line or a line or two but here we're talking about quite a few. :-)